build/Dockerfile-aws

```diff
@@ -1,5 +1,5 @@
 FROM frolvlad/alpine-glibc:alpine-3.15_glibc-2.34
-RUN apk add --no-cache ca-certificates postgresql-client curl 
+RUN apk add --no-cache ca-certificates postgresql-client curl bash
```

Comment:
minor: to end up with less overall image size and layers it would be safe to do:
```
RUN apk add --no-cache \
    ca-certificates \
    postgresql-client \
    curl \
    bash \
  && apk upgrade --no-cache
```

Reply:
Good idea, need to merge a bit urgently. Will do this in a follow up PR.